ethportal-api/src/types/content_key.rs

```diff
@@ -222,9 +202,8 @@ mod test {
 
     #[test]
     fn block_header() {
-        let expected_content_key =
-            hex::decode("00d1c390624d3bd4e409a61a858e5dcc5517729a9170d014a6c96530d64dd8621d")
-                .unwrap();
+        const KEY_STR: &str = "00d1c390624d3bd4e409a61a858e5dcc5517729a9170d014a6c96530d64dd8621d";
```

Comment:
Let's add `0x` prefixes to all these values and replace all usage of the `hex` crate for decoding below with the `hex_decode` util. Which, I'm now realizing is not available inside `ethportal-api` ... so my personal preference would be just to copy and paste the `hex_decode` & `hex_encode` utils from `trin_core` into a `ethportal::utils::bytes` file so they're available to use here

Reply:
Completed:
- Copy utils to `ethportal-api`
- Replace use of `hex::*` with native version
- Update test values to comply